docs/examples/input-number/custom.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <el-input-number v-model="num" />
+  <el-input-number v-model="num">
+    <template #prepend>
+      <el-icon>
+        <ArrowDown />
+      </el-icon>
+    </template>
+    <template #append>
+      <el-icon>
+        <ArrowUp />
+      </el-icon>
+    </template>
+  </el-input-number>
+  <br />
+  <el-input-number v-model="num" controls-position="right" />
+  <el-input-number v-model="num" controls-position="right">
+    <template #prepend>
+      <el-icon>
+        <Minus />
+      </el-icon>
+    </template>
+    <template #append>
+      <el-icon>
+        <Plus />
+      </el-icon>
+    </template>
+  </el-input-number>
```

Comment:
It seems that the layout here can also be adjusted, it currently looks too compact.

Reply:
![image](https://github.com/element-plus/element-plus/assets/69418751/5dbddca3-0d88-46db-bb68-1378f17bf120)